src/react-components/room/ChatSidebarContainer.js

```diff
@@ -282,14 +301,21 @@ export function ChatSidebarContainer({ scene, canSpawnMessages, presences, occup
   return (
     <ChatSidebar onClose={onClose}>
       <ChatMessageList ref={listRef} onScroll={onScrollList}>
-        {messageGroups.map(({ id, systemMessage, ...rest }) => {
+        {messageGroups.map(entry => {
+          const { id, systemMessage, type } = entry;
           if (systemMessage) {
-            return <SystemMessage key={id} {...rest} />;
+            return <SystemMessage key={id} {...entry} />;
           } else {
-            return <ChatMessageGroup key={id} {...rest} />;
+            if (type === "permission") {
+              return !isMod && <PermissionMessageGroup key={id} {...entry} />;
```

Comment:
Again should mods really not see these messages?

Reply:
Updated.